examples/stable-diffusion/README.md

```diff
@@ -296,8 +295,25 @@ python text_to_image_generation.py \
     --bf16
 ```
 
-> For improved performance of the SD3 pipeline on Gaudi, it is recommended to configure the environment
-> by setting PT_HPU_MAX_COMPOUND_OP_SIZE to 1.
+Here is how to generate SD3 images with a single prompt with FP8 precision:
```

Comment:
Can you also comment here that we first need to run "measure" quant mode to generate stats for quantization?